services/gundeck/src/Gundeck/API.hs

```diff
@@ -20,259 +20,19 @@ module Gundeck.API
-      .&. header "Z-User"
-      .&. opt (query "since")
-      .&. opt (query "client")
-      .&. def (unsafeRange 1000) (query "size")
-  document "GET" "fetchNotifications" $ do
-    summary "Fetch notifications"
-    parameter Query "since" bytes' $ do
-      optional
-      description "Only return notifications more recent than this."
-    parameter Query "client" bytes' $ do
-      optional
-      description "Only return notifications targeted at the given client."
-    parameter Query "size" (int32 (Swagger.def 1000)) $ do
-      optional
-      description "Maximum number of notifications to return."
-    returns (ref Model.notificationList)
-    response 200 "Notification list" end
-    errorResponse' notificationNotFound Model.notificationList
-  get "/notifications/:id" (continue getByIdH) $
-    accept "application" "json"
-      .&. header "Z-User"
-      .&. capture "id"
-      .&. opt (query "client")
-  document "GET" "getNotification" $ do
-    summary "Fetch a notification by ID."
-    parameter Query "id" bytes' $
-      description "Notification ID"
-    parameter Query "client" bytes' $ do
-      optional
-      description "Only return notifications targeted at the given client."
-    returns (ref Model.notification)
-    response 200 "Notification found" end
-    errorResponse notificationNotFound
-  get "/notifications/last" (continue getLastH) $
-    accept "application" "json"
-      .&. header "Z-User"
-      .&. opt (query "client")
-  document "GET" "getLastNotification" $ do
-    summary "Fetch the last notification."
-    parameter Query "client" bytes' $ do
-      optional
-      description "Only return the last notification targeted at the given client."
-    returns (ref Model.notification)
-    response 200 "Notification found" end
-    errorResponse notificationNotFound
-  -- Presence API ----------------------------------------------------------
-
-  get "/i/presences/:uid" (continue Presence.list) $
-    param "uid" .&. accept "application" "json"
-  get "/i/presences" (continue Presence.listAll) $
-    param "ids" .&. accept "application" "json"
-  post "/i/presences" (continue Presence.add) $
-    request .&. accept "application" "json"
-  delete "/i/presences/:uid/devices/:did/cannons/:cannon" (continue Presence.remove) $
-    param "uid" .&. param "did" .&. param "cannon"
-  -- User-Client API -------------------------------------------------------
-
-  delete "/i/clients/:cid" (continue unregisterClientH) $
-    header "Z-User" .&. param "cid"
-  delete "/i/user" (continue removeUserH) $
-    header "Z-User"
-  -- Docs ------------------------------------------------------------------
-
-  get "/push/api-docs" (continue docsH) $
-    query "base_url" .&. accept "application" "json"
-  -- Status & Monitoring ---------------------------------------------------
-
+  Public.sitemap
+  Public.apiDocs
+  Internal.sitemap
   head "/i/status" (continue $ const (return empty)) true
```

Comment:
Why did you leave the `/i/status` out from the internal endpoints?

Reply:
They seemed very generic to me, in that every service has some common non-domain-related endpoints. Seems to me like they should be separated from the domain endpoints and could even be abstracted out for re-use (although in this case there's no big benefit for just 2 trivial endpoints).

Reply:
Also, in some services we don't have an `Internal` module, so we would have to create another module just for the status endpoint or it will be inconsistent.

Reply:
Okay, I'll move them to the other internal endpoints.

Reply:
Thanks!